<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reestr_ttn</class>
 <widget class="QMainWindow" name="reestr_ttn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>525</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>АРМ Сбит </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>90</y>
      <width>710</width>
      <height>381</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>РЕЄСТР НАКЛАДНИХ</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>721</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>563</x>
      <y>50</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>52</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Пошук по №</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>РІК</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>1990</number>
     </property>
     <property name="maximum">
      <number>2099</number>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>МІСЯЦЬ</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="maxVisibleItems">
      <number>12</number>
     </property>
     <item>
      <property name="text">
       <string>Січень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лютий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Березень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Квітень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Травень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Червень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Липень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Серпень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вересень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Жовтень</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Листопад</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Грудень</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>F2 Перегляд</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>F9 Друк</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>F10 Видалити</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>F5 Редаг.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>480</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>F4 Ввід</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
